src/components/Dashboard/TodoList.vue

```diff
@@ -684,6 +719,8 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
       }
     }
 
+    await this.loadAffiliationInvitationsTodo()
```

Comment:
Do you actually want to load affiliation invitations before the other tasks?

Reply:
Moved the call to be before adding other items.